mtags/src/main/scala-2/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -274,11 +276,22 @@ class CompletionProvider(
       def isNotLocalForwardReference: Boolean =
         !head.sym.isLocalToBlock ||
           !head.sym.pos.isAfter(pos)
+      def isFileAmmoniteCompletion() = {
```

Comment:
This removes any default `$file` completions coming from how the packages in the generated files are organized.